packages/core/src/new-api/stored-deployment-serializer.ts

```diff
@@ -312,10 +334,48 @@ export class StoredDeploymentSerializer {
     return arg;
   }
 
+  private static _serializeRuntimeValue(
+    arg: RuntimeValue
+  ): SerializedRuntimeValue {
+    if (arg.type === RuntimeValueType.ACCOUNT) {
+      return this._serializeAccountRuntimeValue(arg);
+    }
+
+    return this._serializeModuleParamterRuntimeValue(arg);
+  }
+
+  private static _serializeAccountRuntimeValue(
+    arg: AccountRuntimeValue
+  ): SerializedAccountRuntimeValue {
+    return { _kind: "AccountRuntimeValue", accountIndex: arg.accountIndex };
+  }
+
+  private static _serializeModuleParamterRuntimeValue(
+    arg: ModuleParameterRuntimeValue<ModuleParameterType>
+  ): SerializedModuleParameterRuntimeValue {
+    return {
+      _kind: "ModuleParameterRuntimeValue",
+      name: arg.name,
+      defaultValue:
+        arg.defaultValue === undefined
+          ? undefined
+          : this._jsonStringifyWithBigint(arg.defaultValue),
```

Comment:
The serialization of the this default value has to be fairly generic. I decided to use a JSON string, as it can handle everything we need, except `bigint`. For those, I used [a replacer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#replacer).